### PR TITLE
ENH: draw motors

### PR DESCRIPTION
**Pull request by Gui-FernandesBR**
## Pull request type

- [ ] Code changes (bugfix, features)
- [ ] Code maintenance (refactoring, formatting, tests)
- [ ] ReadMe, Docs and GitHub updates
- [ ] Other (please describe):

## Checklist

- [ ] Tests for the changes have been added (if needed)
- [ ] Docs have been reviewed and added / updated
- [ ] Lint (`black rocketpy/ tests/`) has passed locally 
- [ ] All tests (`pytest --runslow`) have passed locally

## Current behavior

Enter text here...

## New behavior

Enter text here...

## Breaking change

- [ ] Yes
- [ ] No

## Additional information
No hynrids so far

### CODE REVIEW

File: rocketpy/plots/liquid_motor_plots.py
Comment (phmbressan) [inline rocketpy/plots/liquid_motor_plots.py:5]: I don't know if it is only an overlook or the code lacks some calls, but two imports were not being used here. The fix is on the suggestion.

```suggestion
import numpy as np
import matplotlib.pyplot as plt
```

File: rocketpy/plots/liquid_motor_plots.py
Comment (phmbressan) [inline rocketpy/plots/liquid_motor_plots.py:304]: Is this code finished? What these comments should indicate?

File: rocketpy/plots/rocket_plots.py
Comment (phmbressan) [inline rocketpy/plots/rocket_plots.py:352]: Only to make sure I understand the situation, the reason the draw is broken up in pieces here instead of directly calling the `motor.draw()` is because the final result would be too cluttered?

File: rocketpy/plots/solid_motor_plots.py
Comment (phmbressan) [inline rocketpy/plots/solid_motor_plots.py:389]: Lost comments.
```suggestion
```

File: rocketpy/plots/solid_motor_plots.py
Comment (phmbressan) [inline rocketpy/plots/solid_motor_plots.py:397]: One more lost comment.

```suggestion
```

File: rocketpy/plots/tank_plots.py
Comment (phmbressan) [inline rocketpy/plots/tank_plots.py:4]: Same comment about unused imports. Suggestion:

```suggestion
import matplotlib.pyplot as plt
import numpy as np
from matplotlib.patches import Polygon
```

**Review by phmbressan — COMMENTED** [review]
Excellent plots, these are some small comments I have while skimming through the code. I will do a deeper review/test the functionalities later.

**Comment by Gui-FernandesBR** [comment]
Current status: Waiting for @phmbressan deeper review, as commented before. And also one more review would be nice.

**Comment by phmbressan** [comment]
Are you planning to implement `HybridMotor` drawings in this PR or it will come in a future PR?

**Comment by Gui-FernandesBR** [comment]
> Are you planning to implement `HybridMotor` drawings in this PR or it will come in a future PR?

Good question. I'm currently not working on this. Implementing draws for hybrid motors would depend a lot on te current implementation of liquid and solid, therefore I'd say it is better to finish up those first.

To be honest the code is far from ideal. Both rocket draw and motors draw would benefit a lot from code refactors.

Let's see how these go in our roadmap. Code suggestions are still welcome in the meantime.

**Comment by MateusStano** [comment]
@Gui-FernandesBR @phmbressan need a re-review here!

Hybrid drawings have been added, and all bugs (I could find) have been fixed

I highly recommend messing with the positions and coordinate systems when testing this

File: rocketpy/plots/rocket_plots.py
Comment (phmbressan) [inline rocketpy/plots/rocket_plots.py:343]: Small code suggestion that does not affect funcionality: we could avoid the many `isinstances` and repeated code by catching `AttributeError` exceptions if the tank does not have any grains for example.

**Review by phmbressan — APPROVED** [review]
Great work here from @Gui-FernandesBR and @MateusStano. The drawings are incredible, really useful to have a confirmation of distances and positioning of motor components.

**Comment by Gui-FernandesBR** [comment]
I can't approve because I'm the one who created the PR. But everything is fine now. I added a few docstrings that were lacking. 

This PR improves the overall rocketpy experience, this is a quite innovative feature that is being added.

Ready to merge if passing on tests

File: rocketpy/plots/rocket_plots.py
Comment (Gui-FernandesBR) [reply rocketpy/plots/rocket_plots.py:343]: I agree with the suggestion but I think we can refactor this later without causing any trouble. The code is working fine as it is right now, and any user/developer will understand that we created different conditions for each type of motor.

Thanks anyway.

**Pull request merged**
